### PR TITLE
change the Navbar hover effect on Navlinks

### PR DESCRIPTION
**Pull request by sohamsshah**
## Related issue
- Relates to the UI revamping of the present website. Not from any present issue. (Will work on them too sooner)

## Proposed changes
- CSS Animation of Middle-outwards hover effect on the Navlinks to make it look more beautiful.

## Additional Information
- Will work on the About Us Page after doing a bit of UI research, and also in the overall project soon. This is my first contribution to Elemento :) Looking forward for support and guidance!

## Screenshots
Since this is an animation, Screenshot isn't possible.

## Summary
In a nutshell when the user will hover on the Navlinks, there will be a middle-outwards animation purple underline effect which is sleeker and simple than before with better spacing. It is a small change, but good for a start. More to come! Thank you #Elemento team! Also I have commented the original code out, just for reference in future re-usablility.

### CODE REVIEW

**Comment by sohamsshah** [comment]
@AasthaGithub Can you please help me out why my all checks have failed? :(

**Comment by AasthaGithub** [comment]
The greetings check has failed to Github Actions Configuration, nothing to worry abt code. So, am merging now.
